app/routers/grade.py

```
@@ -0,0 +1,49 @@
+from app.core.security import get_current_user
+from app.schemas.grade import GradeRequest, GradingResultResponse
+from app.services import grade_service
+
+router = APIRouter(prefix="/answers")
+
+
+@router.post("/{answersheet_id}/grade", status_code=status.HTTP_201_CREATED)
+async def grade_quiz(
+    answersheet_id: int,
+    grade_request: GradeRequest,
+    db: AsyncSession = Depends(get_db),
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)